API.md

```
@@ -508,3 +508,25 @@ Allows calling a curated raw Blockscout API endpoint for advanced or chain-speci
 ```bash
 curl "http://127.0.0.1:8000/v1/direct_api_call?chain_id=1&endpoint_path=/api/v2/proxy/account-abstraction/operations&query_params[sender]=0x91f51371D33e4E50e838057E8045265372f8d448"
 ```
+
+### Report Tool Usage (`report_tool_usage`)
+
+Receive an anonymous tool usage report from a community-run server.
+
+`POST /v1/report_tool_usage`
+
+**Parameters**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)